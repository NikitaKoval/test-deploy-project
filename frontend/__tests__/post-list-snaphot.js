import React from 'react'
import renderer from 'react-test-renderer'
import PostsList from '../components/posts-list'

it('renders post list unchanged', () => {
    const postList = [
        {
            "id": 3,
            "text": "zzz",
            "timestamp": "2021-10-18T07:46:17.476166Z"
        },
        {
            "id": 2,
            "text": "bbb",
            "timestamp": "2021-10-18T07:45:35.627924Z"
        },
        {
            "id": 1,
            "text": "aaa",
            "timestamp": "2021-10-18T07:30:13.217922Z"
        }
    ]
    const tree = renderer.create(<PostsList posts={postList} />).toJSON()
    expect(tree).toMatchSnapshot()
})