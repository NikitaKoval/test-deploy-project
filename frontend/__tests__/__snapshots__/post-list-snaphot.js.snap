// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders post list unchanged 1`] = `
<ul>
  <li>
    zzz
    <span>
       
      <small>
        10/18/2021, 7:46:17 AM
      </small>
    </span>
  </li>
  <li>
    bbb
    <span>
       
      <small>
        10/18/2021, 7:45:35 AM
      </small>
    </span>
  </li>
  <li>
    aaa
    <span>
       
      <small>
        10/18/2021, 7:30:13 AM
      </small>
    </span>
  </li>
</ul>
`;
